iib/web/models.py

```diff
@@ -1131,6 +1131,11 @@ def to_json(self, verbose=True):
             self.from_bundle_image_resolved, 'pull_specification', None
         )
         rv['organization'] = self.organization
+        if current_app.config['IIB_REQUEST_RELATED_BUNDLES_DIR']:
+            rv['related_bundles'] = {
+                'expiration': self.temporary_data_expiration.isoformat() + 'Z',
+                'url': url_for('.get_related_bundle_images', request_id=self.id, _external=True),
```

Comment:
Can you add a comment here on how the URL will look like?

Reply:
What do you mean? 
Check this out: https://flask.palletsprojects.com/en/2.0.x/api/#flask.url_for